docker-black/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-alpine3.7
+
+WORKDIR /app
+RUN pip install black
```

Comment:
pin a version

Reply:
Right now it's on alpha stage but ok, I'll pin it to 18.4a2, which is the one I'm using right now.